common/src/main/java/com/wynntils/features/inventory/InventoryEmeraldCountFeature.java

```diff
@@ -83,6 +85,12 @@ public void onContainerRender(ContainerRenderEvent event) {
                 ? containerScreen.leftPos + 2
                 : screen.width - containerScreen.leftPos - 2;
 
+        if (Models.Container.getCurrentContainer() instanceof PersonalStorageContainer
+                && Managers.Feature.getFeatureInstance(PersonalStorageUtilitiesFeature.class)
+                        .isEnabled()) {
+            textureX -= 10 + Texture.BANK_PANEL.width();
+        }
```

Comment:
Don't like this but not sure what the best alternative is, maybe changing the texture locations for banks to be above on either side of the bank type name?

Reply:
Hm, yes. This is more of a texture issue than a code issue, so there is no point in holding this PR for it,